docs/release-notes.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 11
+id: release-notes
+description: prophecy monthly release notes
+tags: [release notes, changelog]
+title: Release Notes
+sidebar_label: Release Notes
+---
+
+### Release 2.6.0.0 February 2022
```

Comment:
Might be better as:

```
## 2.6.0.0 (February 2023)
```

(Level 2 header instead of 3)

Reply:
The release should be level 2, everything else below should be 3 or 4, so it'll all be grouped under the release